app/src/main/java/org/fossasia/susi/ai/chat/ChatPresenter.kt

```diff
@@ -66,13 +69,10 @@ class ChatPresenter(context: Context) :
     var id: Long = 0
     var identifier: String = ""
     var tableItem: TableItem? = null
-    private val planHandler: Handler = Handler()
-    lateinit var youtubeVid: IYoutubeVid
-    private var isPlanAction = false
-    private var planQueryAnswer: String = " "
+    private val youtubeVid: IYoutubeVid = YoutubeVid(context)
     private var textToSpeech: TextToSpeech? = null
-    private var isVideo = false
-    private lateinit var planVideoSusiResponse: SusiResponse
+    private val context: Context = context
```

Comment:
Why?

Reply:
I was failing to access the ```context``` from any functions. So, I declared it as a global variable, and this value has been used then.